kernel-headers/CMakeLists.txt

```diff
@@ -26,6 +26,11 @@ publish_internal_headers(rdma
   rdma/vmw_pvrdma-abi.h
   )
 
+publish_internal_headers(rdma/hfi
+  rdma/hfi/hfi1_ioctl.h
+  rdma/hfi/hfi1_user.h
+  )
+
```

Comment:
This script oopsie needs to be fixed before this can be applied

Reply:
Done.